include/fg/defines.h

```diff
@@ -114,6 +114,17 @@ enum ColorMap {
     FG_BLUEMAP  = 6                         ///< Blue color map
 };
 
+enum Color {
```

Comment:
Please change the names to follow other enum's convention of `FG_` prefix.
Instead of RED color, just use word `Crimson` i think the `(r,g,b)` values are `(220, 20, 60)`